### PR TITLE
combine ibm, aws into tabs, add tabset for getting started

### PR DESCRIPTION
**Pull request by devpramod**

### CODE REVIEW

**Review by tomlenth — APPROVED** [review]
looks great, thx!

File: getting-started/README.md [hunk 1]
```diff
@@ -11,8 +12,41 @@ Before moving forward, it's important to familiarize yourself with two key eleme
...
+:sync: AWS
+
+1. Navigate to [AWS console](https://console.aws.amazon.com/console/home) – Search EC2 in the search bar and select it. Click the `Launch Instance` button highlighted in orange. 
+
+2. Provide a name to the VM. 
+
+3. In Quick Start, select the base OS as Ubuntu (ami-id : ami-04dd23e62ed049936). 
+
+4. Select an Instance type that is based on Intel hardware. 
+>**Note**: We recommend selecting a m7i.4xlarge or larger instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor. For more information on virtual servers on AWS visit [here](https://aws.amazon.com/intel/).
```
Comment (louie-tsai) [inline getting-started/README.md:34]: might be good to mention needed disk size. doesn't hurt to ask for 100GB

File: getting-started/README.md [hunk 2]
```diff
@@ -11,8 +12,41 @@ Before moving forward, it's important to familiarize yourself with two key eleme
...
+2. Provide a name to the VM. 
+
+3. In Quick Start, select the base OS as Ubuntu (ami-id : ami-04dd23e62ed049936). 
+
+4. Select an Instance type that is based on Intel hardware. 
+>**Note**: We recommend selecting a m7i.4xlarge or larger instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor. For more information on virtual servers on AWS visit [here](https://aws.amazon.com/intel/). 
+
+5. Next, create a new key pair by giving it a name or select one from the existing key pair. 
+
+6. Under Network settings select an existing security group. If there is none, create a new one allowing SSH, HTTP and HTTPS traffic .
```
Comment (louie-tsai) [inline getting-started/README.md:38]:
```suggestion
6. Under Network settings select an existing security group. If there is none, create a new one allowing SSH,  and HTTP traffic . 

```

File: getting-started/README.md [hunk 3]
```diff
@@ -62,11 +100,12 @@ Now we can start the services
...
 docker compose up -d
 ```
-It takes a few minutes for the services to start. Check the logs for the services to ensure that ChatQnA is running.
+>**Note**: It takes a few minutes for the services to start. Check the logs for the services to ensure that ChatQnA is running before proceeding further.
+
 For example to check the logs for the `tgi-service`:
 
 ```bash
-docker compose logs tgi-service | grep Connected
+docker container logs tgi-service | grep Connected
```
Comment (louie-tsai) [inline getting-started/README.md:108]: is that "container" or "compose"?

File: getting-started/README.md [hunk 4]
```diff
@@ -92,6 +131,32 @@ Run `docker ps -a` as an additional check to verify that all the services are ru
...
+
+:sync-group: Cloud Service Providers
+
+:::{tab-item} AWS
+
+:sync: AWS
+
+You can interact with ChatQnA via a browser interface: 
+
+* To view the ChatQnA interface, open a browser and navigate to the UI by inserting your externally facing IP address in the following: `http://{public_ip}:80’.
```
Comment (louie-tsai) [inline getting-started/README.md:145]:
```suggestion
* To view the ChatQnA interface, open a browser and navigate to the UI by inserting your public facing IP address in the following: `http://{public_ip}:80’. 

```

File: getting-started/README.md [hunk 5]
```diff
@@ -92,6 +131,32 @@ Run `docker ps -a` as an additional check to verify that all the services are ru
...
+:sync: AWS
+
+You can interact with ChatQnA via a browser interface: 
+
+* To view the ChatQnA interface, open a browser and navigate to the UI by inserting your externally facing IP address in the following: `http://{public_ip}:80’. 
+>**Note**: If you are having issue accessing the UI, you need to add an inbound rule in the security group. 
+
+* Look up Security Groups in the search bar and select the security group used when creating the instance. 
+* Click on the Edit inbound rules on the right side of the window. 
+* Select Add rule at the bottom, and create a rule with type as Custom TCP , port range as 80 and source as 0.0.0.0/0
```
Comment (louie-tsai) [inline getting-started/README.md:150]: you probably will be removed automatically by corporate security group if you open port 80 to all.....   65535 port for all case.   you might just mention put " select a port safe to open to public. ex: 65535"

**Review by louie-tsai — COMMENTED** [review]
looks good. minor comments

**Comment by louie-tsai** [comment]
@devpramod   do you need me to add GCP contents?

File: getting-started/README.md (hunk 4 above)
Comment (louie-tsai) [reply getting-started/README.md:145]: that is the term used by AWS

File: getting-started/README.md (hunk 5 above)
Comment (aice-support) [reply getting-started/README.md:150]: also, you might need to mention how to change NGINX port accordingly via environment variable.

File: getting-started/README.md (hunk 2 above)
Comment (louie-tsai) [reply getting-started/README.md:38]: we don't use https in ChatQnA.

**Review by yinghu5 — APPROVED** [review]
nice to have

**Comment by NeoZhangJianyu** [comment]
@devpramod 
Have you verified the format of html after convert the md to html locally?
There are code for sphinx grammar.

File: getting-started/README.md (hunk 3 above)
Comment (srinarayan-srikanthan) [reply getting-started/README.md:111]: resolved

File: getting-started/README.md (hunk 5 above)
Comment (louie-tsai) [reply getting-started/README.md:176]: please also try on external IP under your security group. no worries if it also works for external IP.

**Pull request merged**